src/web/components/Input/RadioInput.stories.tsx

```diff
@@ -1,19 +1,30 @@
 import { ComponentMeta, ComponentStory } from '@storybook/react';
+import { FormProvider, useForm } from 'react-hook-form';
 
-import { Form } from '../Core/Form';
 import { RadioInput } from './RadioInput';
 import { Option } from './SelectInput';
 
+import '../Core/Form.scss';
+
 export default {
   title: 'Inputs/Radio',
   component: RadioInput,
 } as ComponentMeta<typeof RadioInput>;
 
 const Template: ComponentStory<typeof RadioInput> = (args) => {
+  const formMethods = useForm();
+  const { handleSubmit } = formMethods;
   return (
-    <Form onSubmit={() => {}}>
-      <RadioInput {...args} inputName='radio' />{' '}
-    </Form>
+    <FormProvider {...formMethods}>
+      <form onSubmit={handleSubmit(() => {})}>
+        <RadioInput {...args} inputName='radio' />{' '}
+        <div className='form-footer'>
+          <button type='submit' className='primary-button'>
+            Submit
+          </button>
+        </div>
```

Comment:
I see a lot of repetition of these 5 lines throughout - I think it might be worth pulling this out into a SubmitButton component. You can have it take the same props as a button and pass them all to the button with the spread operator `...` to keep it just as flexible as an actual button.

Reply:
Created FormSubmitButton component